src/contactsmodel.h

```diff
@@ -0,0 +1,61 @@
+/*
+    Copyright (C) 2020 Sebastian J. Wolf and other contributors
+
+    This file is part of Fernschreiber.
+
+    Fernschreiber is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Fernschreiber is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Fernschreiber. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONTACTSMODEL_H
+#define CONTACTSMODEL_H
+
+#include <QAbstractListModel>
+#include <QVariantList>
+#include <QSqlDatabase>
+#include <QSqlQuery>
+
+#include "tdlibwrapper.h"
+
+class ContactsModel : public QAbstractListModel
+{
+    Q_OBJECT
+public:
+    ContactsModel(TDLibWrapper *tdLibWrapper, QObject *parent = nullptr);
+
+    virtual int rowCount(const QModelIndex &) const;
+    virtual QVariant data(const QModelIndex &index, int role) const;
```

Comment:
It's really really really really strange to see a list model that doesn't define any roles. Like, `firstName`, `lastName` and such. It's like a database that defines a single column called `data` and stores everything in that one column.

Reply:
Which would of course work just fine - why would anyone need more than one column in a database, right? :) 

However, that would significantly limit options for all kinds of optimizations of updates, queries and other operations on that poor database. Same thing here. Yes, one role is enough, just like one column is enough for a database.

Reply:
I was simply the easiest way of getting things done. In this simple thingy I don't really see the need to introduce additional roles, but of course it can be done... But wouldn't that be something that can be considered upfront overhead design? ;) Jokes aside, let me see if I find some time to implement roles here as well...

Reply:
Yep, found some time. :)